about.ftd

```diff
@@ -1,8 +1,6 @@
-
-
-
--- ft.page:
 
+-- ft.doc-page:
+toc: $ft.toc
```

Comment:
@sourabh-garg can we specify the toc in `doc-page` component inside `Jupiter-theme` directly?

```
toc: $ft.toc
```

Seems unnecessary

Reply:
We want to have a default toc in doc-page, also give the option to override that toc. Default feature is getting fixed
by Arpita. Will simplify the code once it's deployed